src/constants/object.js

```diff
@@ -0,0 +1,38 @@
+export const ObjectTypes = {
```

Comment:
i think this was moved to https://github.com/CSCfi/metadata-submitter-frontend/blob/develop/src/constants/wizardObject.js with this PR: https://github.com/CSCfi/metadata-submitter-frontend/pull/180 and other files from `constants` folder ... maybe needs to be rebased on `develop` ?

Reply:
Hmm this is weird since I've merged `develop` and on my local there are only:
![image](https://user-images.githubusercontent.com/48789543/110581878-02513d00-8174-11eb-93d0-cf7b24e3a84c.png)

Reply:
sounds weird, from what I have in this branch:
![Screenshot 2021-03-10 at 9 02 53](https://user-images.githubusercontent.com/23532204/110589796-983e9500-817f-11eb-9b1b-a71a0b7469b7.png)

Reply:
Deleted local branch and pulled from Git. I can see these extra files now.

Reply:
I stand corrected, might not have been moved but split, but Hang has it correct there.

Reply:
I'll removed extra files on my local and will commit on next push.